src/lib/export_backup.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+
+def write_file(output_file, data, data_format='text', heading=None, mode=0o600, exist_ok=True):
+  ''' write text or binary data to file, create file with proper permissions '''
```

Comment:
We use black formatting which will reformat the above two lines... it prefers double quotes and the 96 char function declaration will be split over multiple lines... I can pair with you on installing the relevant pre-commit hooks to the repo perhaps.

Reply:
i am down. i think i've tried `black` w/o success, but that was in Opstools repo.